capstone/src/main/webapp/index.html

```diff
@@ -31,13 +36,13 @@ <h2>Top Negative Associations:</h2>
             <li>Thing 3</li>
           </ul>
         </div>
-        <div class="stats-divider"></div>
         <div id="second-stats" class="container second-stats">
```

Comment:
For these id tags, you might want to make it more descriptive now that you're replacing the placeholders.

`<div id="sentiment-pie-stats"...`